### PR TITLE
Issue-226: (Docs) Moved Deployment Policy FAQ entry

### LINKED ISSUE #226: Documentation📄: Move Deployment Policy FAQ entry
### What is the current documentation state?

A recent commit #225 added a FAQ entry which addressed #114

This additional FAQ about deployment policies seems misplaced however. That's a low level detail that might be better off in the
"Deployment Policy Use Cases" page at
https://github.com/open-horizon/open-horizon.github.io/blob/master/docs/using_edge_services/policy_user_cases.md
which lands here:
https://open-horizon.github.io/docs/using_edge_services/policy_user_cases.html

### Where is this stated?

_No response_

### Why do you want to improve the statement?

_No response_

### Proposed Statement

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

## Description

Moved Deployment Policy FAQ entry to to "Deployment Policy Use Cases" page

Fixes #226 

## Type of change

docs and code 

Please delete options that are not relevant.

- [x] This change requires a documentation update

## How Has This Been Tested?

tested locally on server

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: docs/using_edge_services/policy_user_cases.md [hunk 1]
```diff
@@ -60,6 +60,27 @@ Administrators can create a deployment policy, and {{site.data.keyword.edge_notm
 
 Because deployment policies capture the more dynamic, business-like service properties and constraints, they are expected to change more often than service policy. Their lifecycle is independent from the service they refer to, which gives the policy administrator the ability to state a specific service version or a version range. {{site.data.keyword.edge_notm}} then merges service policy and deployment policy, and then attempts to find nodes whose policy is compatible with that.
 
+## How do you update the constraints of an existing deployment policy with the hzn cli?
```
Comment (joewxboy) [inline docs/using_edge_services/policy_user_cases.md:63]: Simplify title to "Deployment policy updating"

**Review by joewxboy — CHANGES_REQUESTED** [review]
See single inline comment.

File: docs/using_edge_services/policy_user_cases.md (hunk 1 above)
Comment (johnwalicki) [reply docs/using_edge_services/policy_user_cases.md:63]: This additional information should be moved to the ## Constraints section.

**Review by johnwalicki — CHANGES_REQUESTED** [review]

**Review by johnwalicki — APPROVED** [review]
This is great.  Thanks for moving the deployment policy update details to the relevant section.  Thanks also for the markdown style cleanup.

**Comment by johnwalicki** [comment]
@joewxboy is on vacation so I'm going to merge this in his absence.

**Pull request merged**